Exploration Queries/SecurityEvent/ParentProcessesOnHost.txt

```diff
@@ -9,9 +9,8 @@
 // DataSources: #SecurityEvent
 // Tactics: #Persistence, #Discovery, #LateralMovement, #Collection
 
-let GetParentProcessesOnHost = (v_Host:string){
+let GetParentProcessesOnHost = (v_Host_HostName:string){
```

Comment:
agreed on the following naming for function parameters for clarity
**v_Entity_Field**